.angulardoc.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Please remove this file. It's GUI specific, I think.

Reply:
Sure